src/components/ComplaintDetail/ComplaintDetailBody.js

```diff
@@ -168,30 +168,26 @@ export const ComplaintDetailBody = ({ data, error, id }) => {
           <br />
         </div>
         <div className="card-right layout-column">
-          {!!data.timely && (
-            <>
-              <h4 tabIndex="0">Timely response?</h4>
-              <CompanyTimely value={data.timely} />
-              <br />
-            </>
-          )}
-          {!!data.company_response && (
-            <>
-              <h4 tabIndex="0">Company response to consumer</h4>
-              <span className="body-copy" tabIndex="0">
-                {data.company_response}
-              </span>
-              <br />
-            </>
-          )}
-          {!!data.company_public_response && (
-            <>
-              <h4 tabIndex="0">Company public response</h4>
-              <span className="body-copy" tabIndex="0">
-                {data.company_public_response}
-              </span>
-            </>
-          )}
+          <>
+            <h4 tabIndex="0">Timely response?</h4>
+            <CompanyTimely value={data.timely} />
+            <br />
+          </>
+          <>
+            <h4 tabIndex="0">Company response to consumer</h4>
+            <span className="body-copy" tabIndex="0">
+              {data.company_response ? data.company_response : 'N/A'}
+            </span>
+            <br />
+          </>
+          <>
+            <h4 tabIndex="0">Company public response</h4>
+            <span className="body-copy" tabIndex="0">
+              {data.company_public_response
+                ? data.company_public_response
+                : 'N/A'}
+            </span>
+          </>
```

Comment:
You can probably get ready of these react fragment tags `<></>` now that there's no more conditional rendering.